src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -99,8 +103,18 @@ public CommandResult execute(Model model) throws CommandException {
         model.addEntity(toAdd);
 
         if (toAdd instanceof Body) {
-            NotifCommand notifCommand = new NotifCommand(new Notif((Body) toAdd), 5, TimeUnit.SECONDS);
+            Body body = (Body) toAdd;
+            NotifCommand notifCommand = new NotifCommand(new Notif(body), 5, TimeUnit.SECONDS);
```

Comment:
I think abstracting out 5 would be good, especially when allowing the user to change the timing!